src/img_web/app/migrations/0001_initial.py

```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8 -*-
 import datetime
-from south.db import db
-from south.v2 import SchemaMigration
-from django.db import models
+from django import db
+from django.db import migrations, models
 
 
-class Migration(SchemaMigration):
+class Migration(migrations.Migration):
```

Comment:
Migrations can't be ported from South to Django built-in like this. You need to remove the old migrations and run manage.py makemigrations.

If you run migrate on empty DB with migrations like these, you'll see that it does not create required tables as the South migration format is not the same as Django built-in one.